### PR TITLE
Complete query ops

### LINKED ISSUE #12: implement users_repo.go

### LINKED ISSUE #13: implement skills_repo.go

### LINKED ISSUE #14: implement milestones_repo.go

### LINKED ISSUE #15: implement jobs_repo.go

### LINKED ISSUE #16: implement discussions_repo.go

### LINKED ISSUE #17: implement applications_repo.go

### PR DESCRIPTION
**Pull request by Vulfpeck**
Fixes #12 
Fixes #13 
Fixes #14 
Fixes #15 
Fixes #16 
Fixes #17

### CODE REVIEW

**Review by AJaxRUN — APPROVED** [review]

File: graph/resolver/milestone_resolver.go [hunk 1]
```diff
@@ -2,26 +2,40 @@ package resolver
...
+	skills, err := r.SkillsRepo.GetByMilestoneId(obj.ID)
+	if err != nil {
+		return nil, err
+	}
+	var result []*gqlmodel.Skill
+	for _, skill := range skills {
+		var gqlSkill gqlmodel.Skill
+		gqlSkill.MapDbToGql(*skill)
+		result = append(result, &gqlSkill)
+	}
```
Comment (AJaxRUN) [inline graph/resolver/milestone_resolver.go:38]: The function is incomplete, has to return

File: graph/resolver/milestone_resolver.go [hunk 2]
```diff
@@ -2,26 +2,40 @@ package resolver
...
 
-//TODO: Should not exist, or find a better way to implement
-func (m milestonesResolver) TotalCount(ctx context.Context, obj *model.Milestones) (*int, error) {
```
Comment (AJaxRUN) [inline graph/resolver/milestone_resolver.go:24]: func (m *milestoneResolver) typo

File: graph/resolver/milestone_resolver.go (hunk 1 above)
Comment (Vulfpeck) [reply graph/resolver/milestone_resolver.go:38]: Ahahaha. The return statement is right below this comment lmao

File: graph/resolver/milestone_resolver.go (hunk 2 above)
Comment (Vulfpeck) [reply graph/resolver/milestone_resolver.go:24]: It's a part of Milestones and not Milestone, Milestones has a list of Milestone. Also, removed code

**Pull request merged**